ci/credentials.yml.example

```diff
@@ -95,12 +95,14 @@ aws_external_s3_tfstate_bucket: EXTERNAL_TERRAFORM_BUCKET
 cdn_broker_username_staging: cdn-broker-staging
 cdn_broker_bucket_staging: cdn-broker-le-verify-staging
 cdn_broker_cloudfront_prefix_staging: cg-staging/*
-cdn_broker_hosted_zone_staging:
+cdn_broker_hosted_zone_staging: ZZZZZZZZZZZZZZ
```

Comment:
As an aside, this is why I think example credentials are rarely valuable--our variable names should be descriptive enough in most cases, and if they're not, better to rename than add examples.